plugins/tiddlywiki/markdown/files/remarkable-katex-license.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
IMO we should add here: for: `remarkable-katex.js`

Reply:
Or better: Add the reference to this plugin to the $:/plugins/tiddlywiki/markdown/readme  tiddler.

Reply:
Mentioned the plugin usage in https://github.com/Jermolene/TiddlyWiki5/pull/5846/commits/fe013616f3ea8cdbc4621237aa75bde2fb461363

but since Remarkable's license wasn't mentioned there I didn't mention remarkable-katex's license either.  Did you mean I should mention the license there?

Reply:
It would be great to bring in Remarkable's license, it's an oversight that it wasn't already included

Reply:
Sure thing: https://github.com/Jermolene/TiddlyWiki5/pull/5846/commits/8b31c8567f9278206485708ca6babad1d3239adb